test/unit/sanitizer/_geo_reverse.js

```diff
@@ -61,27 +61,88 @@ module.exports.tests.warning_situations = (test, common) => {
+  // value and they are simply stripped out by parseFloat() with warning.
+  test('boundary.circle.radius must be a positive number', (t) => {
+    const raw = {
+      'point.lat': '12.121212',
+      'point.lon': '21.212121',
+      'boundary.circle.radius': '1km' // note the 'km'
+    };
+    const clean = {};
+    const errorsAndWarnings = sanitizer.sanitize(raw, clean);
+
+    t.equals(raw['boundary.circle.lat'], 12.121212);
+    t.equals(raw['boundary.circle.lon'], 21.212121);
+    t.equals(raw['boundary.circle.radius'], '1km');
+    t.equals(clean['boundary.circle.lat'], 12.121212);
+    t.equals(clean['boundary.circle.lon'], 21.212121);
+    t.equals(clean['boundary.circle.radius'], 1.0);
+
+    t.deepEquals(errorsAndWarnings, { errors: [], warnings: [] });
+    t.end();
+  });
+
   test('boundary.circle.radius specified in request should override default', (t) => {
     const raw = {
       'point.lat': '12.121212',
       'point.lon': '21.212121',
-      'boundary.circle.radius': '3248732857km' // this will never be the default
+      'boundary.circle.radius': '2'
     };
     const clean = {};
     const errorsAndWarnings = sanitizer.sanitize(raw, clean);
 
     t.equals(raw['boundary.circle.lat'], 12.121212);
     t.equals(raw['boundary.circle.lon'], 21.212121);
-    t.equals(raw['boundary.circle.radius'], '3248732857km');
+    t.equals(raw['boundary.circle.radius'], '2');
     t.equals(clean['boundary.circle.lat'], 12.121212);
     t.equals(clean['boundary.circle.lon'], 21.212121);
-    t.equals(clean['boundary.circle.radius'], 3248732857.0);
+    t.equals(clean['boundary.circle.radius'], 2.0);
 
     t.deepEquals(errorsAndWarnings, { errors: [], warnings: [] });
     t.end();
+  });
+
+  test('boundary.circle.radius specified should be clamped to MAX', (t) => {
+    const raw = {
+      'point.lat': '12.121212',
+      'point.lon': '21.212121',
+      'boundary.circle.radius': '10000'
+    };
+    const clean = {};
+    const errorsAndWarnings = sanitizer.sanitize(raw, clean);
+
+    t.equals(raw['boundary.circle.lat'], 12.121212);
+    t.equals(raw['boundary.circle.lon'], 21.212121);
+    t.equals(raw['boundary.circle.radius'], '10000');
+    t.equals(clean['boundary.circle.lat'], 12.121212);
+    t.equals(clean['boundary.circle.lon'], 21.212121);
+    t.equals(clean['boundary.circle.radius'], 5.0);
 
+    t.deepEquals(errorsAndWarnings, { errors: [], warnings: [] });
+    t.end();
   });
 
+  test('boundary.circle.radius specified should be clamped to MIN', (t) => {
+    const raw = {
+      'point.lat': '12.121212',
+      'point.lon': '21.212121',
+      'boundary.circle.radius': '0.000000000000001'
+    };
+    const clean = {};
+    const errorsAndWarnings = sanitizer.sanitize(raw, clean);
+
+    t.equals(raw['boundary.circle.lat'], 12.121212);
+    t.equals(raw['boundary.circle.lon'], 21.212121);
+    t.equals(raw['boundary.circle.radius'], '0.000000000000001');
```

Comment:
I think you can skip testing the `raw` values, since you specified them above.

IMO it just makes the test longer and it took me a second to realize it was checking the raw value, not clean. At first glance it makes this test look like it doesn't work!

Reply:
Hmm actually no, that's not right, because if so, then the raw values would all be strings 🤷‍♂️ 
Anyway, I just cleaned up the tests by removing the assertions for `$raw`